collector/info_schema_processlist.go

```diff
@@ -26,6 +32,14 @@ var (
 		"collect.info_schema.processlist.min_time",
 		"Minimum time a thread must be in each state to be counted",
 	).Default("0").Int()
+	processesDisableByUserFlag = kingpin.Flag(
+		"collect.info_schema.processlist.disable_processes_by_user",
```

Comment:
I prefer positive flags when possible, to avoid confusing double negatives.

Let's use `collect.info_schema.processlist.processes_by_user`, with a default of true.  Same for by_host.

This way the disable flag will be `--no-collect.info_schema.processlist.processes_by_user`.